### PR TITLE
Init model and add relationship

### PR DESCRIPTION
**Pull request by thanhnn-3239**
## Related Tickets
- [#50939](https://edu-redmine.sun-asterisk.vn/issues/50939)

## WHAT (optional)
- Init model and add relationship

## HOW
- Init model and add relationship theo thiết kế database

## WHY (optional)
- Init model and add relationship

### CODE REVIEW

**Comment by thanhnn-3239** [comment]
ready

**Comment by thanhnn-3239** [comment]
ready

**Comment by thanhnn-3239** [comment]
ready

File: db/migrate/20220711083953_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email, null: false, index: {unique: true}
+      t.string :phone_num,  index: {unique: true}
+      t.string :address
+      t.string :password_digest
+      t.integer :role
```
Comment (ngocvt-0484) [inline db/migrate/20220711083953_create_users.rb]: e set default là role j đi

File: db/migrate/20220711084152_create_authors.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateAuthors < ActiveRecord::Migration[6.1]
+  def change
+    create_table :authors do |t|
+      t.string :name
+      t.string :description
```
Comment (ngocvt-0484) [inline db/migrate/20220711084152_create_authors.rb]: description nên để là text

File: db/migrate/20220711152825_create_orders.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+class CreateOrders < ActiveRecord::Migration[6.1]
+  def change
+    create_table :orders do |t|
+      t.string :name
+      t.string :phone_num
+      t.string :address
+      t.text :note
+      t.integer :status
```
Comment (ngocvt-0484) [inline db/migrate/20220711152825_create_orders.rb]: set default cho status

**Comment by thanhnn-3239** [comment]
ready

**Comment by thanhnn-3239** [comment]
ready

**Pull request merged**